### DISCUSSION TITLE
Yarn installation fails

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#24995] [Codespaces]
<p>My containers based on the javascript-node image fail to build when they try to install Yarn:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e2ef52863307c4790e70a2e8a552f2cb0b0fd48a.png" data-download-href="/uploads/short-url/wnyEypyP927dlBoPOSmfC5Drqhc.png?dl=1" title="Screen Shot 2021-02-03 at 10.31.06 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181088037-785510e9-ef6d-4ad5-b606-5f6df530cbf0.png" alt="Screen Shot 2021-02-03 at 10.31.06 PM" data-base62-sha1="wnyEypyP927dlBoPOSmfC5Drqhc" width="690" height="422" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2ef52863307c4790e70a2e8a552f2cb0b0fd48a_2_690x422.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2ef52863307c4790e70a2e8a552f2cb0b0fd48a_2_1035x633.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2ef52863307c4790e70a2e8a552f2cb0b0fd48a_2_1380x844.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2ef52863307c4790e70a2e8a552f2cb0b0fd48a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-03 at 10.31.06 PM</span><span class="informations">2182×1336 311 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The key needs to be updated, see <a href="https://github.com/yarnpkg/yarn/issues/7866" class="inline-onebox" rel="noopener nofollow ugc">Yarn Debian key expiry date updated (EXPKEYSIG 23E7166788B63E1E) · Issue #7866 · yarnpkg/yarn · GitHub</a>.</p>

**Comment by jkeech** [comment, marked as answer]
<p>Hey <a class="mention" href="/maiertech">@maiertech</a>, thanks for reporting! This is also reported in <a href="https://github.community/t/not-able-to-install-yarn-and-angular-cli-and-create-codespace-container/159239/3" class="inline-onebox">Not able to install yarn and angular cli and create codespace container - #3 by Chuxel</a>. We’re pushing out updated images, but you can also reference a more specific tag instead of latest as a workaround until the image caches are updated everywhere.</p>

**Comment by maiertech** [comment]
<p>Waiting long enough for updated caches helped.</p>
